lib/ueberauth/strategy/auth0.ex

```diff
@@ -123,6 +123,6 @@ defmodule Ueberauth.Strategy.Auth0 do
   end
 
   defp option(conn, key) do
-    Dict.get(options(conn), key, Dict.get(default_options, key))
+    Keyword.get(options(conn), key, Dict.get(default_options(), key))
```

Comment:
Whoops, my bad! Yes, the `default_options()` returns a keyword list. Changes pushed.